composer.json

```diff
@@ -38,10 +41,11 @@
         "http-interop/http-middleware": "^0.4.1",
         "laminas/laminas-coding-standard": "^1.0.0",
```

Comment:
Can you perhaps try and update `phpcs.xml` to match the latest schema anyway? I wonder if it still works with that :thinking:

Reply:
Nice, it's working :)